test/integration/external/staging/electrs.test.ts

```diff
@@ -9,7 +9,8 @@ import { BitcoinCoreClient } from "../../../../src/utils/bitcoin-core-client";
 import { REGTEST_ESPLORA_BASE_PATH } from "../../../../src";
 import { BTCAmount } from "@interlay/monetary-js";
 
-describe("ElectrsAPI testnet", function () {
+// TODO: Unskip tests when the electrs instance is fixed
+describe.skip("ElectrsAPI testnet", function () {
```

Comment:
The instance should be working now (cc: @ns212)

Reply:
I think we should actually remove the testnet tests. They only test the testnet deployment, since the same logic is also tested on the Regtest Electrs instance. This E2E functionality should be checked in DataDog or the Bot instead.